packages/provider/src/schedulers/captchaScheduler.ts

```diff
@@ -36,22 +33,28 @@ export async function storeCaptchasExternally(
 			: defaultSchedule
 		: defaultSchedule;
 
-	const job = new CronJob(cronSchedule, async () => {
-		const taskRunning = await checkIfTaskIsRunning(
-			ScheduledTaskNames.StoreCommitmentsExternal,
-			env.getDb(),
-		);
-		env.logger.info(
-			`${ScheduledTaskNames.StoreCommitmentsExternal} task running: ${taskRunning}`,
+	const job = new CronJob(cronSchedule, () => {
+		console.log(`Creating worker - from main thread: ${threadId}`);
+		const worker = new Worker(
+			path.resolve("../provider/dist/workers/storeCaptchaWorker.js"),
```

Comment:
Is it ok to have `dist` here?

Reply:
Should we not be using `import.meta.module` or something or will this completely break in bundling? Actually, does this work after bundling CLI?